dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/DLCOracle.scala

```diff
@@ -30,16 +30,21 @@ class DLCOracle(private[this] val extPrivateKey: ExtPrivateKeyHardened)(implicit
 
   private val rValAccount: HDAccount = {
     val purpose = conf.kmParams.purpose
-    val coin =
-      HDCoin(purpose, HDCoinType.fromNetwork(conf.network))
+    // We have to use Testnet here because before this was dictated by
```

Comment:
can we add migration code here so we don't have to support this indefinitely? 

Where was this introduced? In #2720 or #2667 ? 

Was this in the latest official release of krystal-bull ? 

This is going to suck to have to maintain forever :/

Reply:
Can we get some unit tests? This seems very concerning that this isn't being caught by tests ?

Reply:
> Where was this introduced? In #2720 or #2667 ?

This was introduced from the beginning :/

> Was this in the latest official release of krystal-bull ?

Yes but not a problem because you can't change the network config

> This is going to suck to have to maintain forever :/

Shouldn't require any changes after this because it is a hard coded value

Reply:
> can we add migration code here so we don't have to support this indefinitely?

done in the app config

Reply:
We are essentially hardcoding a _specific_ network here (Testnet), which isn't really "network-agnostic". 

Perhaps we should just try to allocate a new HDCoinType for the DLC oracle? It seems that might be the best choice we make for the future, while having to live with this bug we currently have to support?

Reply:
> Perhaps we should just try to allocate a new HDCoinType for the DLC oracle? It seems that might be the best choice we make for the future, while having to live with this bug we currently have to support?

We already did that for the purpose, and if we change it, it will break all current oracles.

> We are essentially hardcoding a specific network here (Testnet), which isn't really "network-agnostic".

Kinda, we have to pick one tho so I am just picking the default so we don't break compatibilty